### PR TITLE
Feature/new component logo

### PR DESCRIPTION
**Pull request by DeepaliAgrwl**
Please merge the changes

### CODE REVIEW

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/structure/logo/_cq_dialog/.content.xml [hunk 1]
```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:granite="http://www.adobe.com/jcr/granite/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+    jcr:primaryType="nt:unstructured"
+    jcr:title="Logo"
+    sling:resourceType="cq/gui/components/authoring/dialog"
+    extraClientlibs="[core.wcm.components.image.v2.editor]">
+    <content
+        granite:class="cmp-image__editor"
+        jcr:primaryType="nt:unstructured"
+        sling:resourceType="granite/ui/components/coral/foundation/container">
```
Comment (rakeshkumar-verma) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/structure/logo/_cq_dialog/.content.xml]: @DeepaliAgrwl  Why you need this dialog here?

Comment (DeepaliAgrwl) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/structure/logo/_cq_dialog/.content.xml]: Not required.removed

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/structure/logo/clientlib/less/logo.less [hunk 2]
```diff
@@ -0,0 +1,6 @@
+.logo-container{
+
+display : flex;
```
Comment (rakeshkumar-verma) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/structure/logo/clientlib/less/logo.less]: @DeepaliAgrwl  Remove unused styling

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/structure/logo/.content.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:Component"
```
Comment (rakeshkumar-verma) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/structure/logo/.content.xml:3]: @DeepaliAgrwl  also updated header component policy

**Pull request merged**